### PR TITLE
added tags to mplot3d examples

### PR DESCRIPTION
**Pull request by farquh**
## PR summary
Improves documentation of mplot3d examples by adding tags.

## PR checklist

- [N/A] "closes #0000" is in the body of the PR description to [link the related issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [N/A] new and changed code is [tested](https://matplotlib.org/devdocs/devel/testing.html)
- [N/A] *Plotting related* features are demonstrated in an [example](https://matplotlib.org/devdocs/devel/document.html#write-examples-and-tutorials)
- [N/A] *New Features* and *API Changes* are noted with a [directive and release note](https://matplotlib.org/devdocs/devel/api_changes.html#announce-changes-deprecations-and-new-features)
- [ ] Documentation complies with [general](https://matplotlib.org/devdocs/devel/document.html#write-rest-pages) and [docstring](https://matplotlib.org/devdocs/devel/document.html#write-docstrings) guidelines

### CODE REVIEW

File: galleries/examples/mplot3d/2dcollections3d.py [hunk 1]
```diff
@@ -46,3 +46,9 @@
 ax.view_init(elev=20., azim=-35, roll=0)
 
 plt.show()
+
+# %%
+# .. tags::
+#    plot-type: 3D, scatter, line
```
Comment (QuLogic) [inline galleries/examples/mplot3d/2dcollections3d.py:52]: These (and elsewhere it occurs) should be separate:
```suggestion
#    plot-type: 3D
#    plot-type: scatter
#    plot-type: line
```

**Review by QuLogic — COMMENTED** [review]
Please also fix the linting error.

**Comment by QuLogic** [comment]
Sorry, I missed it earlier, but every tag list should be comma-separated.

**Comment by farquh** [comment]
> Hi, sorry it took so long to review this. Reviews are mostly fixing formatting - please let me know if you plan to finish this out or if someone should take over. Thanks!

Sorry I missed this and also didn't respond for awhile. Someone else should probably take over. Thank you!

**Comment by rcomer** [comment]
> Someone else should probably take over.

Thanks for letting us know @farquh!

**Comment by QuLogic** [comment]
I rebased and took care of the issues with the tags.

**Review by scottshambaugh — APPROVED** [review]

**Comment by scottshambaugh** [comment]
Thank you @farquh for your work on this - it will be part of the upcoming v3.10 release. Congratulations on your first contribution to matplotlib, and we hope to see you again!

**Pull request merged**